actionview/lib/action_view/cache_expiry.rb

```diff
@@ -42,12 +42,14 @@ def clear_cache
 
     private
       def dirs_to_watch
-        fs_paths = all_view_paths.grep(FileSystemResolver)
```

Comment:
How about we just change this to: `all_view_paths.grep(FileSystemResolver).map!(&:path).tap(&:uniq!).sort!`?

Reply:
That looks good. And the allocation savings are pretty much the same.